Readme fixes (#455)
spatialos/UnrealGDK: README.md

The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin (Unreal documentation)](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.

The GDK offers:

> * **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.

Reviewer: ```suggestion
* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables you to use more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.
```

> * **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.

Reviewer 2: Original:
* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.

->
* Wondering if we say "Unreal game server" it suggests we are on Unreal's servers. We have made a distinction between Unreal servers and SpatialOS servers in the documentation (out of necessity because of the switch).
* "enables usage " is a bit awkward.
* "in a single instance" isn't clear what we mean? A single instance of what? When I get an idea, I can suggest some alternative text.

Also, this text is the same as the welcome/index page - so need to synch up when we have the final copy here.

We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).

## Where to get the GDK and related projects
The GDK and its related projects are available on GitHub.
* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)

> **NOTE:** This link may give you a 404. 

Reviewer: We also mention the 404 issue above (under Unreal Engine changes), but there we link directly to the GDK docs on the main docs site, instead of explaining inline. Do we want to do the same here? (Or explain inline in both places?)

It's a minor point though - probably fine to stay as is.

Author: Yeah not sure to be honest. I'll leave it as is right now.

> **NOTE:** This link may give you a 404. 

Reviewer 2: Think this needs a <br/> to line break it and make it easier to read.
